README.md

```diff
@@ -1,88 +1,130 @@
 # Akka Serialization Helper
 
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.virtuslab/sbt-akka-serialization-helper/badge.svg)](https://maven-badges.herokuapp.com/maven-central/cz.jirutka.rsql/rsql-parser)
+
+
 Serialization toolbox for Akka messages, events and persistent state that helps achieve compile-time guarantee on
 serializability.
 
 ## Install
 
-To install the library use JitPack:
+To use the library use sbt plugin (version equals number on the maven badge):
 
 ```scala
-resolvers += "jitpack".at("https://jitpack.io")
-val commit = "master-SNAPSHOT" //name of a branch with -SNAPSHOT or raw commit hash
+addSbtPlugin("org.virtuslab.ash" % "sbt-akka-serialization-helper" % Version)
+```
+and enable it in target project:
+```scala
+enablePlugins(AkkaSerializationHelperPlugin)
 ```
 
-Then, add one or more of the modules below:
-
-## Modules
-
-The project consists of three modules that are independent of each other, comprising a complete solution together.
+## Features
 
 ### 1. Check For Base Trait 
 
-A Scala compiler plugin that detects messages, events and persistent state, and checks whether they extend the base
+A Scala compiler plugin that detects messages, events and persistent states, and checks whether they extend the base
 trait and report an error when they don't. This ensures that the specified serializer is used by Akka and protects
 against accidental use
 of [Java serialization](https://doc.akka.io/docs/akka/current/serialization.html#java-serialization).
 
-To use, just annotate a base trait with `@org.virtuslab.ash.SerializabilityTrait`:
+To use, annotate a base trait with `@org.virtuslab.ash.SerializabilityTrait`:
 
 ```scala
 @SerializabilityTrait
 trait MySerializable
 ```
 
```

Comment:
Plus maybe a small sample of a situation when an error is reported + the error itself